packages/shared/src/utils/communities.ts

```diff
@@ -1,12 +1,3 @@
-export const GRID_ENABLED_COMMUNITY_ADDRESSES = [
```

Comment:
Clean up this since we enabled this layout to all community

Reply:
good call thank you!